firmware/arduino/chessboard/motorDriver.ino

```diff
@@ -129,13 +129,18 @@ void home()
 // A specific motor is passed to this function since we are only moving one here
 // Returns a 'MovementStatus' code, '0' if successful, varying nonzero values for various error codes
 // For all status codes, check 'MovementStatus' in chessboard.ino
-uint8_t moveStraight(uint8_t motor[], int startCol, int startRow, int endCol, int endRow)
+uint8_t moveStraight(uint8_t motor[], int endCol, int endRow)
 {
   // How many steps per space
   int dir, numSteps, unitSpaces;
   int i;
   int eighthStepsPerPulse;
   int *currentMotorPos;
+  float startCol, startRow;
+
+  // Converts current position to be in terms of unit spaces
```

Comment:
Just make sure to add to your comment explaining what the (stepsPerUnitSpace * 8) means